client/extensions/woocommerce/woocommerce-services/state/label-settings/selectors.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+export const getLabelSettingsForm = ( state, siteId = getSelectedSiteId( state ) ) => {
```

Comment:
getLabelSettingsForm could use `get` with default value set to `null` so you could later just do:
`return form && form.data`